### PR TITLE
adding mission groups

### PR DESCRIPTION
**Pull request by svetlichnaya**
first pass at adding mission groups!

### CODE REVIEW

File: spaceship/models/mission.py [hunk 1]
```diff
@@ -22,3 +22,24 @@ class Mission(BaseModel):
...
+  def is_over(self):
+    now = pendulum.now('UTC')
+    mission_end_time = self.started_at.add(weeks=self.duration_in_weeks)
+    return now >= mission_end_time
+
+  @hybrid_property
+  def is_running(self):
+    now = pendulum.now('UTC')
+    mission_end_time = self.started_at.add(weeks=self.duration_in_weeks)
+    return now >= self.started_at and now <= mission_end_time
```
Comment (igorkofman) [inline spaceship/models/mission.py]: does this make missions 8 days long instead of 7?

File: spaceship/models/mission.py [hunk 2]
```diff
@@ -22,3 +22,24 @@ class Mission(BaseModel):
   @hybrid_property
   def is_active(self):
     return self.deleted_at == None
+
+  @hybrid_property
+  def start_time(self):
```
Comment (igorkofman) [inline spaceship/models/mission.py]: maybe worth calling this "formatted_start_time" or "start_time_str"?

**Review by igorkofman — CHANGES_REQUESTED** [review]

File: spaceship/models/mission.py (hunk 1 above)
Comment (jeredw) [reply spaceship/models/mission.py]: should probably be < rather than <=

File: spaceship/models/mission.py [hunk 3]
```diff
@@ -22,3 +22,24 @@ class Mission(BaseModel):
   @hybrid_property
   def is_active(self):
     return self.deleted_at == None
+
+  @hybrid_property
```
Comment (igor47) [inline spaceship/models/mission.py]: i think you want this (and others here) to just be an `@property` -- `@hybrid_property` is for things that can be used to build DB queries. http://docs.peewee-orm.com/en/latest/peewee/playhouse.html#hybrid-attributes

**Review by igor47 — APPROVED** [review]
lgtm except for the one comment

File: spaceship/models/mission.py (hunk 3 above)
Comment (svetlichnaya) [reply spaceship/models/mission.py]: ok! sounds like we can always convert it back to hybrid if we need those queries.

File: spaceship/models/mission.py (hunk 2 above)
Comment (svetlichnaya) [reply spaceship/models/mission.py]: yes, good point!

File: spaceship/models/mission.py (hunk 1 above)
Comment (svetlichnaya) [reply spaceship/models/mission.py]: < makes sense to me--8 days does not, should be adding exactly 7*24hours. Tested with the "?now=YYYY-MM-DD parameter in the url.

**Comment by svetlichnaya** [comment]
hey @jeredw and @igorkofman, I made the changes requested--should be good to go! i want to make the rest of this page nicer :)

**Review by igorkofman — APPROVED** [review]

**Pull request merged**